ansible_inventory_server/jujurest.py

```diff
@@ -22,27 +22,19 @@
 from ansible_inventory_server.utils import (ApiRequestHandler,
                                             filter_ip_addresses)
 
-from ansible_inventory_server import settings
-
 
 async def get_juju_model(parameters):
     """Returns a new juju.model.Model(), using the specified connection
     parameters. Returns None on error"""
-    cacert = parameters.get('juju', {}).get('cacert')
-    if cacert is None:
-        with open(settings.CACERT_PATH, 'r') as fin:
-            cacert = fin.read()
 
     model = Model()
     try:
         await model.connect(
-            cacert=cacert,
+            cacert=parameters['juju']['cacert'],
```

Comment:
This is a low-key breaking change, since we now require all these options be passed in every request. Update relevant docs in https://github.com/grnet/ansible-inventory-server/blob/master/API.md#passing-parameters and https://github.com/grnet/ansible-inventory-server/blob/master/API.md#parameters

Reply:
done